test/modules/MetricsValidator/src/tests/ValidateNumberOfMessagesSent.cs

```diff
@@ -1,6 +1,6 @@
 // Copyright (c) Microsoft. All rights reserved.
 
-namespace MetricsValidator
+namespace MetricsValidator.Tests
```

Comment:
namespace <==> folder

Reply:
Oh, should I have the src in the namespace? All the other test modules ignore the src folder for their namespace.

Reply:
Nope, you're good